app/frontend/src/components/forms/ExportSubmissions.vue

```diff
@@ -1,269 +1,309 @@
-        >
-        <v-card-text>
-          <hr
-            style="
-              height: 2px;
-              border: none;
-              background-color: #707070c1;
-              margin-top: 5px;
-            "
-          />
-
-          <v-row>
-            <v-col>
-              <p class="subTitleObjectStyle">Select the submission date</p>
-              <v-radio-group v-model="dataFields" hide-details="auto">
-                <v-radio label="All data/fields" :value="false">
-                  <template v-slot:label>
-                    <span class="radioboxLabelStyle">All data/fields</span>
-                  </template>
-                </v-radio>
-                <!--<v-radio label="Select Date Range" :value="true">
-                  <template v-slot:label>
-                    <span class="radioboxLabelStyle">Selected data/fields</span>
-                    <v-icon class="ml-3" color="#003366"> view_week</v-icon>
-                  </template>
-                </v-radio>
-                -->
-              </v-radio-group>
-            </v-col>
-          </v-row>
-          <v-row>
-            <v-col>
-              <p class="subTitleObjectStyle">Select the submission date</p>
-              <v-radio-group v-model="dateRange" hide-details="auto">
-                <v-radio label="All" :value="false">
-                  <template v-slot:label>
-                    <span class="radioboxLabelStyle">All</span>
-                  </template>
-                </v-radio>
-                <v-radio label="Select Date range" :value="true">
-                  <template v-slot:label>
-                    <span class="radioboxLabelStyle">Select date range</span>
-                  </template>
-                </v-radio>
-              </v-radio-group>
-            </v-col>
-          </v-row>
-
-          <div v-if="dateRange">
-            <v-row>
-              <v-col cols="12" sm="6" offset-sm="0" offset-md="1" md="4">
-                <v-menu
-                  v-model="startDateMenu"
-                  data-test="menu-form-startDate"
-                  :close-on-content-click="true"
-                  :nudge-right="40"
-                  transition="scale-transition"
-                  offset-y
-                  min-width="290px"
-                >
-                  <template v-slot:activator="{ on }">
-                    <label>From</label>
-                    <v-text-field
-                      v-model="startDate"
-                      placeholder="yyyy-mm-dd"
-                      append-icon="event"
-                      v-on:click:append="startDateMenu = true"
-                      readonly
+  <div>
+    <v-row class="mt-6" no-gutters>
+      <v-col>
+        <v-row>
+          <v-col cols="12" sm="6" order="2" order-sm="1">
+            <h1>Export Submissions to File</h1>
+          </v-col>
+          <v-col class="text-right" cols="12" sm="6" order="1" order-sm="2">
+            <span v-if="canViewSubmissions">
```

Comment:
I think this conditional is unnecessary as this page should only be accessible if you were a submission reviewer.